### ISSUE TITLE
Red warning on every launch?

### ISSUE THREAD

**Issue by gingerbeardman** [GitHawkApp/GitHawk#2742]
I see one of these in every launch. 

The message in it varies but nothing ever seems to actually be wrong?

The closest to wrong was when I had to force refresh my bookmarks. 

![GitHawk Upload by gingerbeardman](https://i.imgur.com/ygkTbcV.jpg)

- What request?
- What timeout, I just launched the app?
- Why do I need to know?
- Should I be worried?

<details>
<summary>Bug Report Dump (Auto-generated)</summary>
<pre>
Version 1.28.0 (1554002345)
Device: iPhone 6s (iOS 12.2)
TestFlight: true
</pre>
</details>

**Comment by j-f1** [comment]
If you tap anywhere on the screen with three fingers, you can open FLEX. 🌐 menu → 📡 Network History should have a list of requests. Are any of them failing?

**Comment by gingerbeardman** [comment]
I've switched on FLEX network debugging and will report back.

**Comment by gingerbeardman** [comment]
Here are some captures. 

1.
![GitHawk Upload by gingerbeardman](https://i.imgur.com/Qw3vxBL.jpg)
+
![GitHawk Upload by gingerbeardman](https://i.imgur.com/8BLaVrX.jpg)

2, 3, 4
![GitHawk Upload by gingerbeardman](https://i.imgur.com/AMJFB6E.jpg)
+
![GitHawk Upload by gingerbeardman](https://i.imgur.com/7WV4RS4.jpg)
+
![GitHawk Upload by gingerbeardman](https://i.imgur.com/FJD6Ban.jpg)
+
![GitHawk Upload by gingerbeardman](https://i.imgur.com/xobSqvp.jpg)

**Comment by gingerbeardman** [comment]
More

![GitHawk Upload by gingerbeardman](https://i.imgur.com/X009IMB.jpg)

![GitHawk Upload by gingerbeardman](https://i.imgur.com/Ymejkmy.jpg)

![GitHawk Upload by gingerbeardman](https://i.imgur.com/Kqptauf.jpg)

**Comment by BasThomas** [comment]
I haven't had any issues lately, and it seems like most of these are timeouts... not sure what we can do about them. 😕

**Comment by gingerbeardman** [comment]
These all appear when i open the app after some period (days) of not using it.

But a timeout from what? Are they network request from the previous (very old) session?

**Comment by BasThomas** [comment]
It looks like most of them are many seconds long, yeah. I've seen 6000, 1800... not sure exactly what's going on in those cases, however 😕

**Comment by gingerbeardman** [comment]
How odd. 

I won't post any more examples unless instructed.

**Comment by BasThomas** [comment]
Thanks a lot for sharing these though, it helps!

I just saw that in all of these cases you're in Airplane mode (but connected to WiFi); any idea if that may or may not impact it?

**Comment by gingerbeardman** [comment]
I live in an area with no cell coverage (yes, in 2019!) so that's just how I roll most of the time to prevent the iPhone looking for cell signal that will never be there. 

I've not noticed it affect any apps, so I'd be surprised if GitHawk was the first

**Comment by BasThomas** [comment]
Right. That said, it might still be that GitHawk is the first. As far as I know there's no logic for showing only specific network related errors, so it might be the first...

Thanks again for the insight!

**Comment by gingerbeardman** [comment]
I'm out and about running errands, with airplane mode deactivated and iPhone on cellular data, and I just got one. 

![GitHawk Upload by gingerbeardman](https://i.imgur.com/ElHUBnh.jpg)

![GitHawk Upload by gingerbeardman](https://i.imgur.com/QapOo3m.jpg)

![GitHawk Upload by gingerbeardman](https://i.imgur.com/Ufh5gwh.jpg)

![GitHawk Upload by gingerbeardman](https://i.imgur.com/K9Tnh4x.jpg)

**Comment by gingerbeardman** [comment]
Could it be related to the app getting closed when not in use, due to low memory?

![GitHawk Upload by gingerbeardman](https://i.imgur.com/nJCEqTA.jpg)
